### DISCUSSION TITLE
Stutter flying over every waypoint

### DISCUSSION THREAD

**Discussion by DecentAngle** [flybywiresim/aircraft#6617] [Help]
When flying in North America the sim seems to stutter when passing over waypoints along the flight plan, strangely this does not happen when I fly in the UK and rest of Europe. This started happening after the custom FMS integration, thought maybe I could find someone having similar problems somewhere on the multitude of forums out there but sadly after months of finding nothing I thought I would create an account here to inquire.

I fully removed flight simulator about a month ago and only just reinstalled the sim about a week ago, made sure to go through and delete every trace of it, used revo uninstaller to even go as far as removing any reg keys, but no matter what I do this problem still persists.

**Comment by jcc1953** [comment]
Hello,
I had this problem, for the first time, yesterday, flying above Europe with the last dev version (2022-02-02-02T15:48:27). I went back to the previous dev version (2022-02-01T19:12:14) and no more problem.
